src/utils/hapticFeedbacks.js

```diff
@@ -0,0 +1,19 @@
+/* @flow strict-local */
+
+import { Platform } from 'react-native';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+
+/**
+ * Haptic feedback will trigger if iOS >= 10 OR Device >= iPhone6s.
+ *
+ * In Android feedback trigger for all devices irrespective of system settings
+ */
+export const longPressHapticFeedback = () => {
+  const hapticTriggerType = Platform.select({
+    ios: 'selection',
+    android: 'longPress',
+  });
+  ReactNativeHapticFeedback.trigger(hapticTriggerType, {
+    ignoreAndroidSystemSettings: true,
```

Comment:
We shouldn't ignore the system settings here.

Reply:
>should we favour this "if Haptic is disabled in the Android system settings, allow ignoring the setting and trigger haptic feedback"?
I aksed about that before :)
well, I will remove that, should i remove and push -f or wait for request that we did on upstream?

Reply:
Go ahead and push with the fix. I don't see a reason to wait on upstream, since the helper function that you wrote gets around the problem.